_episodes/15-practices.md

```diff
@@ -227,6 +184,67 @@ Peak rule.
     While it has been criticized for not strongly predicting what's remembered,
     it's always worth trying to end a lesson on a high note.
 
+Instructor guides.
+:   Each Software and Data Carpentry lesson has an instructor's guide, with notes 
```

Comment:
Several of the DC lessons don't yet have instructors' guides. The ones that do have the guides linked from our main lessons page.